analyzer/pkg/rpm/rpm.go

```diff
@@ -104,9 +106,13 @@ func (a rpmPkgAnalyzer) parsePkgInfo(rc io.Reader) ([]types.Package, []string, e
 			}
 		}
 
-		files, err := pkg.InstalledFiles()
-		if err != nil {
-			return nil, nil, xerrors.Errorf("unable to get installed files: %w", err)
+		//check if the package is preinstalled on the system
```

Comment:
I think this is not a pre-installed package. You may install a new package provided by the vendor later.

```suggestion
		// check if the package is vendor-provided
```